src/settings/settingsmanager.cpp

```diff
@@ -27,6 +26,15 @@ constexpr const char* SETTINGS_ORGANIZATION_NAME = "mozilla";
 constexpr const char* SETTINGS_ORGANIZATION_NAME = "mozilla_testing";
 #endif
```

Comment:
I thin this doesn't need to be different anymore now that the app name is different.